addons/web_editor/static/src/components/media_dialog/video_selector.js

```diff
@@ -104,7 +104,7 @@ export class VideoSelector extends Component {
             if (this.props.media) {
                 const src = this.props.media.dataset.oeExpression || this.props.media.dataset.src || (this.props.media.tagName === 'IFRAME' && this.props.media.getAttribute('src')) || '';
                 if (src) {
-                    this.state.urlInput = src;
+                    this.state.urlInput = 'https:'+src;
```

Comment:
Hello @ksbh-odoo looks good, just we are following double quotes everywhere so adapt that and we are good to go.